desktop-widgets/tab-widgets/TabDivePhotos.cpp

```diff
@@ -106,6 +109,36 @@ void TabDivePhotos::recalculateSelectedThumbnails()
 	Thumbnailer::instance()->calculateThumbnails(getSelectedFilenames());
 }
 
+void TabDivePhotos::saveSubtitles()
+{
+	QVector<QString> selectedPhotos;
+	if (!ui->photosView->selectionModel()->hasSelection())
+		return;
+	QModelIndexList indexes = ui->photosView->selectionModel()->selectedRows();
+	if (indexes.count() == 0)
+		indexes = ui->photosView->selectionModel()->selectedIndexes();
+	selectedPhotos.reserve(indexes.count());
+	for (const auto &photo: indexes) {
```

Comment:
Ok - now I see what this is trying to do. I'll have to remember how we store date & length information for the added media. I'm sure we have that information, so scanning the files seems pointless.